### DISCUSSION TITLE
Discord link

### DISCUSSION THREAD

**Discussion by pawlyk** [Lightning-AI/dl-fundamentals#28] [General discussions]
Hi! 
I know it can't be related to the course but I have not found an appropriate way to inform you about that. 
The link to the Discord community is broken.

**Comment by rasbt** [comment]
Thanks for the note, where would that link be so that I can forward the website to our webdev?

**Reply by pawlyk** [reply]
I have found at least two places:

- in the header of the site, Community menu, link to Discord;
- in the footer of [AI Education](https://lightning.ai/pages/ai-education/) and [Lightning Events](https://lightning.ai/pages/events/) pages, link to Discord.

Maybe there are also other places.

**Reply by rasbt** [reply]
Thanks a lot for the note! I will forward this and hope to get this fixed. Much appreciated!!

**Reply by rasbt** [reply]
@pawlyk Just heard back and multiple people confirmed that the link works. Maybe it was a temporary issue at that time that has been fixed in the meantime. Could you try again? Please don't hesitate to reach out and complain if things still don't work on your end!

**Reply by pawlyk** [reply]
@rasbt Thank you for the quick reply. Yes, I have checked and it works. It seems that this is related to a classical cache issue) 
Thank you and sorry for bothering you.

**Reply by rasbt** [reply]
Thanks for double checking, glad it works now!
